packages/zero-cache/src/services/change-streamer/change-streamer-http.ts

```diff
@@ -155,7 +155,12 @@ export class ChangeStreamerHttpClient implements ChangeStreamer {
     if (!address) {
       throw new Error(`no change-streamer is running`);
     }
-    const uri = new URL(path, `http://${address}/`);
+    const uri = new URL(
```

Comment:
not tested currently. Not sure on the best approach given the current tests rely running a local server. Maybe need to start a https server too and disable certificate validation?